money.go

```diff
@@ -87,12 +94,44 @@ func New(amount int64, code string) *Money {
 }
 
 // NewFromFloat creates and returns new instance of Money from a float64.
-// Always rounding trailing decimals down.
+// ONLY USE IN TESTS!!
```

Comment:
Rather than saying "ONLY USE IN TESTS" I would add an example of problems we can have using floats.

Reply:
Makes sense, I'll add the example in the above comment.